components/combobox/src/auro-combobox.js

```diff
@@ -820,12 +820,23 @@ export class AuroCombobox extends AuroElement {
         }
       }
 
-      if (evt.key === 'Tab') {
-        setTimeout(() => {
-          if (document.activeElement !== this) {
-            this.hideBib();
+      if (evt.key === 'Tab' && this.dropdown.isPopoverVisible) {
+        if (this.dropdown.isBibFullscreen) {
+
+          // when focus is on the input, move focus back to close button with Tab key
+          if (document.activeElement.shadowRoot.activeElement === this.inputInBib &&
+            this.inputInBib.shadowRoot.activeElement.tagName !== 'INPUT' &&
+            !evt.shiftKey) {
+            evt.preventDefault();
+            this.dropdown.focus();
           }
-        }, 0);
+        } else {
+          setTimeout(() => {
+            if (document.activeElement !== this) {
+              this.hideBib();
+            }
+          }, 0);
+        }
```

Comment:
When I run this code in combobox tab key just repeats across close, input, clear and is trapped in that cycle.